pkg/kubelet/pluginmanager/operationexecutor/operation_executor.go

```diff
@@ -96,10 +99,9 @@ func (oe *operationExecutor) RegisterPlugin(
 	socketPath string,
 	timestamp time.Time,
 	pluginHandlers map[string]cache.PluginHandler,
```

Comment:
Remove `pluginHandlers map[string]cache.PluginHandler` from `RegisterPlugin(`

Reply:
pluginHandlers is used by registration (see operationGenerator#GenerateRegisterPluginFunc):
```
		handler, ok := pluginHandlers[infoResp.Type]
```
the handler depends on the type of the response.
Can you elaborate how to retrieve the handler if the map is dropped ?

Reply:
Once you implement the suggestion in https://github.com/kubernetes/kubernetes/pull/89934/files#r405081120 you will no longer need it here.

Reply:
pluginHandlers can be removed from GenerateUnregisterPluginFunc without side effect (since it is not used by unregistration). Looking at the code for registration, it seems pluginHandlers is still needed.

Reply:
Sorry you're right, I was thinking of `pathToHandlers` not `pluginHandlers`.

Reply:
Please let me know whether pluginHandlers should be removed from GenerateUnregisterPluginFunc. Keeping it as is would make the signatures symmetrical.

Reply:
@saad-ali 
Let me know if there is remaining comment to be addressed.

Reply:
@saad-ali 
Gentle ping for review.